test/uk/gov/hmrc/pillar2/generators/Generators.scala

```diff
@@ -31,18 +31,20 @@
  */
 
 package uk.gov.hmrc.pillar2.generators
-
-import org.scalacheck.Arbitrary._
-import org.scalacheck.Gen._
-import org.scalacheck.{Arbitrary, Gen, Shrink}
+// scalafix:off
+import org.scalacheck.{Arbitrary, Gen}
+// scalafix:on
```

Comment:
That's a fun compiler crash! Bumping to scala 3.3.7 seems to fix it. Any objections to the most recent LTS?